.env.tmpl

```
@@ -24,4 +24,5 @@ RABBITMQ_PORT={{ .RabbitMQPort }}
 RABBITMQ_DEFAULT_USER={{ .RabbitMQDefaultUser }}
 RABBITMQ_DEFAULT_PASS={{ .RabbitMQDefaultPass }}
 RABBITMQ_MINION_USER={{ .RabbitMQMinionUser }}
-RABBITMQ_MINION_PASS={{ .RabbitMQMinionPass }}
\ No newline at end of file
+RABBITMQ_MINION_PASS={{ .RabbitMQMinionPass }}
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RABBITMQ_MINION_PASS key should go before the RABBITMQ_MINION_USER key

(UnorderedKey)